xml/admin_ceph_upgrade.xml

```diff
@@ -1397,6 +1397,146 @@ data:
    </step>
   </procedure>
  </sect1>
+
+ <sect1 xml:id="filestore2bluestore">
+  <title>OSD Migration to &bluestore;</title>
+  <para>
+   OSD &bluestore; is a new back end for the OSD daemons. It is the default
+   option since &productname; &productnumber;. Compared to FileStore, which
+   stores objects as files in an XFS file system, &bluestore; can deliver
+   increased performance because it stores objects directly on the underlying
+   block device. &bluestore; also enables other features, such as built-in
+   compression and EC overwrites, that are unavailable with FileStore.
+  </para>
+  <para>
+   Specifically for &bluestore;, an OSD has a 'wal' (Write Ahead Log) device
+   and a 'db' (RocksDB database) device. The RocksDB database holds the
+   metadata for a &bluestore; OSD. These two devices will reside on the same
+   device as an OSD by default, but either can be placed on faster/different
+   media.
+  </para>
+  <para>
+   In SES5, both FileStore and &bluestore; are supported and it is possible
+   for FileStore and &bluestore; OSDs to co-exist in a single cluster. During
+   the SUSE Enterprise Storage upgrade procedure, FileStore OSDs are not
+   automatically converted to &bluestore;. Be aware that the
+   &bluestore;-specific features will not be available on OSDs that have not
+   been migrated to &bluestore;.
+  </para>
+  <para>
+   Before converting to &bluestore;, the OSDs need to be running &productname;
+   &productnumber;. The conversion is a slow process as all data gets
```

Comment:
Does this need to be an explicit, specific version, too?

Reply:
Are you thinking that some version of SES6 did not support bluestore and filestore?